cli/cmd/vulnerabilty_exceptions.go

```diff
@@ -0,0 +1,354 @@
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/core"
+	"github.com/lacework/go-sdk/api"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// vulnerability-exceptions command is used to manage lacework vulnerability exceptions
+	vulnerabilityExceptionCommand = &cobra.Command{
+		Use:     "vulnerability-exception",
+		Aliases: []string{"vulnerability-exceptions", "ve", "vuln-exception", "vuln-exceptions"},
+		Short:   "Manage vulnerability exceptions",
+		Long:    "Manage vulnerability exceptions to control and customize your alert profile for hosts and containers.",
+	}
+
+	// list command is used to list all lacework vulnerability exceptions
+	vulnerabilityExceptionListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all vulnerability exceptions",
+		Long:    "List all vulnerability exceptions configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			vulnerabilityExceptions, err := cli.LwApi.V2.VulnerabilityExceptions.List()
+			if err != nil {
+				return errors.Wrap(err, "unable to get vulnerability exceptions")
+			}
+			if len(vulnerabilityExceptions.Data) == 0 {
+				msg := `There are no vulnerability exceptions configured in your account.
+
+Get started by integrating your vulnerability exceptions to manage alerting using the command:
+
+    lacework vulnerability-exception create
+
+If you prefer to configure vulnerability exceptions via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Vulnerabilities > Exceptions.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(vulnerabilityExceptions)
+			}
+
+			var rows [][]string
+			for _, vuln := range vulnerabilityExceptions.Data {
+				rows = append(rows, []string{vuln.Guid, vuln.ExceptionName, vuln.ExceptionType, vuln.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "TYPE", "STATE"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework vulnerability exception by id
+	vulnerabilityExceptionShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Get vulnerability exception by id",
+		Long:  "Get a single vulnerability exception by it's vulnerability exception ID.",
```

Comment:
In some docs you refer to id as `id` and some other places as `ID`.

Which one should we use? 🤔 

```
➜  go-sdk git:(dmurray-lacework/ALLY-752) ✗ git grep Short | grep -i " id"
cli/cmd/alert_rules.go:		Short: "Show an alert rule by id",
cli/cmd/report_rules.go:		Short: "Show a report rule by id",
cli/cmd/resource_groups.go:		Short: "Get resource group by id",
cli/cmd/vuln_host.go:		Short: "List the hosts that contain a specified CVE id in your environment",
cli/cmd/vulnerabilty_exceptions.go:		Short: "Get vulnerability exception by id",
➜  go-sdk git:(dmurray-lacework/ALLY-752) ✗ git grep Long | grep -i " id"
cli/cmd/alert_rules.go:		Long:  "Show a single alert rule by it's ID.",
cli/cmd/alert_rules.go:		Long:  "Delete a single alert rule by it's ID.",
cli/cmd/compliance_azure.go:		Long: `List all Azure subscriptions from the provided Tenant ID.
cli/cmd/compliance_gcp.go:		Long:  `List all GCP projects and organization IDs.`,
cli/cmd/compliance_gcp.go:		Long: `List all GCP projects from the provided organization ID.
cli/cmd/lql_delete.go:		Long: `Delete a single LQL query by providing the query id.
cli/cmd/policy.go:		Long: `Delete a policy by providing the policy id.
cli/cmd/report_rules.go:		Long:  "Show a single report rule by it's id.",
cli/cmd/report_rules.go:		Long:  "Delete a single report rule by it's ID.",
cli/cmd/resource_groups.go:		Long:  "Get a single resource group by it's resource group ID.",
cli/cmd/resource_groups.go:		Long:  "Delete a single resource group by it's resource group ID.",
cli/cmd/vuln_host.go:		Long: `List the hosts that contain a specified CVE id in your environment.
cli/cmd/vulnerabilty_exceptions.go:		Long:  "Get a single vulnerability exception by it's vulnerability exception ID.",
cli/cmd/vulnerabilty_exceptions.go:		Long:  "Delete a single vulnerability exception by it's vulnerability exception ID.",```

Reply:
I prefer `ID`